docs/development/deploying-at-scale/index.md

```diff
@@ -46,3 +46,31 @@ Thanks for flying REANA 🚀
 
 !!! note
     Note that you can deploy REANA in different namespaces by passing `--namespace` to `helm install`. Remember to pass `--create-namespace` if the namespace you want to use does not exist yet. For more information on how to work with namespaces see the [official documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+
+## Advanced deployment scenarios
+
+### High availability
+
+The services running the REANA system are critical for the platform to properly work, therefore it is a good technique to deploy them in different nodes than the ones used to run user workflows. To achieve this:
```

Comment:
REANA infrastructure services are critical ...

... good technique to deploy them in dedicated nodes different from ones used to run user workflows